oss_fuzz.go

```diff
@@ -0,0 +1,43 @@
+//go:build go1.18 || go1.19 || go1.20
+// +build go1.18 go1.19 go1.20
+
+package toml
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+func FuzzToml(data []byte) int {
+	if len(data) >= 10240 {
+		return 0
+	}
+
+	if strings.Contains(string(data), "nan") {
+		return 0
+	}
+
+	var v interface{}
+	err := Unmarshal(data, &v)
+	if err != nil {
+		return 0
+	}
+
+	encoded, err := Marshal(v)
+	if err != nil {
+		return 0
```

Comment:
Should this be a panic? It seems like a piece of that returned by `Unmarshal` should be able to go through `Marshal` without error.